e2e/fixtures/attestations/.bcr/attestations.template.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I don't follow why this is a template to begin with. What's the goal in giving rules authors the affordance to put arbitrary content in this file? Is it so they could also attest to other files they've published to their release that Publish To BCR isn't aware of? 

Like for bazel-lib, what would this template file look like?

Reply:
It's also a way of making the generation of that file optional. It will only be substituted if it exists.

Reply:
Idea from the meeting: if the user has their own attestations, they can provide a json file which is merged with the one generated here. That assumes the schema is a stable public API.

Reply:
Yeah that could work. I think that merging would happen in the workflow. In our last meeting we decided that attestations.json wouldn't be a public template we ask people to fill out, and instead it will be added by the workflow directly. Internally I'm going to keep the template processing of this file and just not document it for now. In the future we could easily support the merging case if it's needed.